libcloud/compute/drivers/vultr.py

```diff
@@ -150,6 +704,19 @@ def require_api_key(self):
             return True
 
 
+class VultrNodeDriverHelper(object):
+    """
+        VultrNode helper class.
+    """
+
+    def handle_extra(self, extra_keys, data):
```

Comment:
This method could also live directly on the ``VultrNodeDriver`` class (perhaps as a class method).

Reply:
I didn't because I have a big plans for improving some cloud providers, especially Vultr. It is just start point.